### ISSUE TITLE
White screen on when i printing

### ISSUE THREAD

**Issue by pablorusso88** [makerbase-mks/MKS-Robin-Nano#38]
hi!

i have a two tree printer on fw 2.0.1

why when I'm printing my screen turns white?
I touch the screen and it doesn't come back, but it does respond because sometimes I touch something and it stops printing

thanks!
i new on this

**Comment by n0xxA1337** [comment]
I have the same problem on Mks_Nano35 V2.0.3.3. Are you printing from Octoprint or the SD card ?

**Comment by pablorusso88** [comment]
SD card, the fw 2.3.1 works fine

**Comment by makerbase-mks** [comment]
You can try to update firmware to latest version

**Comment by wealthybigpenis** [comment]
i am also having this problem with 2.03.5. it only happens about about an hour into printing once the display has gone to sleep. when i touch it to wake it up i have a white screen the rest of the print

**Comment by mks-viva** [comment]
@wealthybigpenis 
Can you provide some picture to me when you printing?

**Comment by wealthybigpenis** [comment]
![20210126_200946](https://user-images.githubusercontent.com/77890325/105932583-ae9af200-6012-11eb-9a1b-5e36480ae391.jpg)
Just a solid white screen. Been printing about 3 hours.

**Comment by mks-viva** [comment]
Is printing still in progress? Just the screen cannot display information?

**Comment by wealthybigpenis** [comment]
Yes the printing continues and finishes however the printer has to be power cycled before the screen will start working again

**Comment by mks-viva** [comment]
What seems to affect the display

**Comment by rubiotel** [comment]
Exactly the same thing happens to me, after 2 or 3 hours of printing, I have to turn off the printer and when I try to load filament (outwards) the Z rises.

**Comment by iMarkus** [comment]
Noticed the same on a TwoTrees Bluer printer with Marlin 2.1.1
Sometimes this happens directly after starting the printing process via OctoPrint.

Printing process is not affected.

**Comment by furkan-o** [comment]
same for 2.0 bugfix release (Marlin 2.0.9.2) by mks and onwards versions by marlin. probably a bug of lvglui and not about fscm cable. couldnt replicate with marlin color ui.
